.huskyrc

```diff
@@ -0,0 +1,5 @@
+{
+  "hooks": {
+    "pre-commit": "lint-staged"
+  }
+}
```

Comment:
I do that too at times. There's a flag (`--no-verify`) that you can use for commits when you don't want to use lint-staged. It'll skip lint-staged.

Reply:
Hmm. Yeah, I might be able to get by with that. I've never worked on a project that ran linters on commit though so I'm not sure. Normally I'd expect that sort of thing to be handled by the CI server.

Reply:
I think the main thing it’s useful for is prettier. Otherwise everyone would have to make sure to run prettier (through an editor plugin, or some other way) on their code before committing. Which undoubtedly will be forgotten sometimes, lead to a failed CI build and then maybe frustrate contributors. Whereas this would automatically work for everyone who did an npm install.

We could always remove eslint and stylelint from lintstaged if they get annoying. It should be helping your workflow, not frustrating it after all 😊